src/Commands/Repos.php

```
@@ -0,0 +1,391 @@
+            label: '🎯 Select a repository to clone:',
+            options: $choices
+        );
+
+        if (confirm(
+            label: "🚀 Clone {$selected}?",
+            default: true
+        )) {
+            $output->writeln("<info>🔄 Cloning {$selected}...</info>");
+
+            $repoName = basename($selected, '.git');
+            exec("git clone {$selected} {$repoName}", $gitOutput, $exitCode);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$gitOutput'. (Unused Code Rules)

(UnusedLocalVariable)